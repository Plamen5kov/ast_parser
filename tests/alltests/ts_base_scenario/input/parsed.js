var MyClass = (function (_super) {
    __extends(MyClass, _super);
    function MyClass() {
        _super.apply(this, arguments);
    }
    MyClass.prototype.MyClassMethod1 = function () {
        console.log("smth");
    };
    MyClass.prototype.MyClassMethod2 = function () {
        console.log("smth");
    };
    MyClass = __decorate([
        JavaProxy("asd.asd.As")
    ], MyClass);
    return MyClass;
})(android.view.ViewGroup);