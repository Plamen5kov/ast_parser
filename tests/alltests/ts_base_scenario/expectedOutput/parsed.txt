Java File: asd.asd.As - Extend Class: android.view.ViewGroup - Overridden Methods: MyClassMethod1,MyClassMethod2